dynamic_object_retrieval/dynamic_object_retrieval/CMakeLists.txt

```diff
@@ -57,24 +61,23 @@ else()
 
     # Define the locations of cereal serialization header files
     set(cereal_include_dir ${CMAKE_CURRENT_SOURCE_DIR}/../k_means_tree/cereal/include)
+
+    # Only compile the stopwatch project if it's not with catkin
     include_directories(${cereal_include_dir})
+    set(stopwatch_library_dir ${stopwatch_dir}/build)
+    # Compile the stopwatch project, needed for this
+    ExternalProject_Add(stopwatch_project
+      DOWNLOAD_COMMAND ""
+      SOURCE_DIR ${stopwatch_include_dir}
+      BINARY_DIR ${stopwatch_library_dir}
+      BUILD_COMMAND ${CMAKE_MAKE_PROGRAM}
+      INSTALL_COMMAND ""
+    )
 endif()
 
 # Eh, we need to somehow build this maybe?
-set(stopwatch_dir ${CMAKE_CURRENT_SOURCE_DIR}/stopwatch)
-set(stopwatch_include_dir ${stopwatch_dir}/src)
-set(stopwatch_library_dir ${stopwatch_dir}/build)
 include_directories(${stopwatch_include_dir})
 
-# Compile the stopwatch project, needed for this
-ExternalProject_Add(stopwatch_project
-  DOWNLOAD_COMMAND ""
-  SOURCE_DIR ${stopwatch_include_dir}
-  BINARY_DIR ${stopwatch_library_dir}
-  BUILD_COMMAND ${CMAKE_MAKE_PROGRAM}
-  INSTALL_COMMAND ""
-)
```

Comment:
@marc-hanheide I don't understand what jenkins is complaining about now. With the project excluded when building with catkin, there should be no builds in the source folder any more. Might there be something lingering on the build server causing the error?